ide-helper.js

```diff
@@ -0,0 +1,11 @@
+/*
+Preferences | Languages & Frameworks | JavaScript | Webpack | webpack configuration file
+jetbrains://WebStorm/settings?name=Languages+%26+Frameworks--JavaScript--Webpack
+*/
+module.exports = {
+  resolve: {
+    alias: {
+      'mastodon': path.resolve(__dirname, 'app/javascript/mastodon'),
```

Comment:
CodeClimate raises an issue here that `path` is not defined, you probablly should add `const path = require('path')` at the top of the file.

Reply:
since it's ESLint error no need to require it, just let linter know it's Node global function